### PR TITLE
EPMRPP-42826 Launch execution and issue statistics on common component

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  const defectTypesChartData = defectDataItems.itemTypes;
+  const columns = [];
+
+  const orderedData = orderedContentFields.map((type) => ({
+    key: type,
+    value: defectTypesChartData[type] || 0,
+  }));
+
+  orderedData.forEach((item) => {
+    contentFields.forEach((field) => {
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js]: Fixed

File: app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/launchExecutionChart/utils.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getChartData } from '../common/config/utils';
+
+export const getColumns = (content) => {
+  const EXECUTIONS = '$executions$';
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/launchExecutionChart/utils.js]: Fixed

File: app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js]: Fixed

File: app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js [hunk 3]
```diff
@@ -48,11 +47,10 @@ export const getColumns = (content, contentFields, { orderedContentFields, defec
   }));
 
   orderedData.forEach((item) => {
-    contentFields.forEach((field) => {
-      if (field === item.key) {
-        columns.push([field, item.value]);
-      }
-    });
+    const findedItem = contentFields.find((field) => field === item.key);
```
Comment (DiscoElevator) [inline app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js]: `foundItem`? Or maybe it's better to give it more descriptive name?

Comment (AmsterGet) [reply app/src/components/widgets/singleLevelWidgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart/utils.js]: Yes, of course! My mistake

**Pull request merged**